modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_3_1/DDMStructureUpgradeProcess.java

```diff
@@ -109,13 +109,15 @@ private boolean _isLabelSet(DDMFormField ddmFormField) {
 
 	private void _upgradeDDMStructure() throws Exception {
 		try (PreparedStatement preparedStatement1 = connection.prepareStatement(
-				"select structureId, definition from DDMStructure where " +
-					"classNameId = ? or classNameId = ? order by createDate");
+				StringBundler.concat(
```

Comment:
Why do you need to use StringBundler.concat()?
Can you remove it, please?

Reply:
I think that the author did this to avoid a error on SF test, but I agree with you and will change their code.

Reply:
If the SF fails because of that, we can maintain :+1: 
If not, it is better to remove it because we only maintain code that we can explain why we added.